app/models/variant.rb

```diff
@@ -8,5 +8,6 @@ module FlexCommerce
   #
   #
   class Variant < FlexCommerceApi::ApiBase
+    has_one :product
```

Comment:
They do appear to work differently to AR. I found that if I used belongs_to, I would get an array back rather than a single product. @garytaylor - this is expected isnt it?